pageserver/src/tenant/remote_timeline_client/download.rs

```diff
@@ -300,7 +308,7 @@ where
             }
             Err(DownloadError::Other(ref err)) => {
                 // Operation failed FAILED_DOWNLOAD_RETRIES times. Time to give up.
-                error!("{description} still failed after {attempts} retries, giving up: {err:?}");
+                warn!("{description} still failed after {attempts} retries, giving up: {err:?}");
```

Comment:
Why not keep it an error?

Reply:
I thought we are moving to the callers issuing the errors and not the "library"